api/app/controllers/review.controller.ts

```diff
@@ -283,125 +244,27 @@ export const getAllReviewsForAdmin = async (req: Request, res: Response) => {
   const userSearch = req.query.user as string;
   const page: number = Number(req.query.page) || DB_START_PAGE;
   const limit: number = Number(req.query.limit) || DB_QUERY_LIMIT;
+  const offset = (page - 1) * limit;
```

Comment:
If page isn't passed, we default to `DB_START_PAGE` which is set to 0 [here](https://github.com/bitcointranscripts/transcription-review-backend/blob/a4f9d489b733901204f0014f307c1a29f9895c4c/api/app/utils/constants.ts#L35) and limit is default of 10. This gives us a negative value of -10. Does this affect the value of the offset since we're passing it directly to the db as value for offset?